clusterloader2/testing/experimental/storage/pod-startup/test-result/volume_podstartup_time_results.yaml

```diff
@@ -0,0 +1,170 @@
+pod-with-volumes-startup-latency-scenarios:
+  volume-type: configmap
```

Comment:
nit: is this a valid yaml?

I think that for slices/arrays, you have to have "-" in yaml. I mean:
- volume-type

[not that this is particularly important]

Reply:
yes that's not valid, I try to create a valid one, thanks for heads up!